### PR TITLE
Added libvorbis

### PR DESCRIPTION
**Pull request by dschreij**

### CODE REVIEW

File: recipes/libvorbis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/libvorbis/meta.yaml]: Please add `toolchain`.

File: recipes/libvorbis/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./configure --prefix=${PREFIX}
```
Comment (jakirkham) [inline recipes/libvorbis/build.sh]: Please add `make`. Also is there a `make check` that we can run?

File: recipes/libvorbis/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required (VERSION 3.1.0)
+project (VORBIS)
```
Comment (jakirkham) [inline recipes/libvorbis/CMakeLists.txt]: Please submit this upstream as well so they can build with CMake too.

Comment (dschreij) [inline recipes/libvorbis/CMakeLists.txt]: The xiph organisation are working on their own CMakeLists.txt (see https://github.com/xiph/vorbis/blob/master/CMakeLists.txt), but they haven't included it with any of their releases yet, so I guess it's still WIP. For future versions of this package, I will use the CMakeLists.txt by the package, but for this version I will stick with what works if that's ok.

Comment (jakirkham) [inline recipes/libvorbis/CMakeLists.txt]: Not a problem. Am just generally encouraging the spirit of collaboration.

File: recipes/libvorbis/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    - test -f $PREFIX/lib/libvorbisfile.dylib  # [osx]
+    - test -f $PREFIX/lib/libvorbis.so  # [linux]
+    - test -f $PREFIX/lib/libvorbisenc.so  # [linux]
+    - test -f $PREFIX/lib/libvorbisfile.so  # [linux]
+    - if not exist %LIBRARY_LIB%\\libvorbis.lib exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\\libvorbis.dll exit 1  # [win]
+
+about:
+  home: https://www.xiph.org/
+  license: Public Domain
```
Comment (jakirkham) [inline recipes/libvorbis/meta.yaml]: Are you sure? Looks like BSD 3-Clause. 😕

Comment (jakirkham) [inline recipes/libvorbis/meta.yaml]: Also can add `license_family: BSD`.

Comment (dschreij) [inline recipes/libvorbis/meta.yaml]: You're right. Fixing this!

**Comment by scopatz** [comment]
Thanks @dschreij!

**Pull request merged**